lblod/app-digitaal-loket
# Add flag to poltiezone & hulpverlenigzone besluitenlijst

## Description

When a submission is send, there is a background job checking if the send submission should be published tot eh publieke besluitendatabank. The prepare-submissions-for-export service is responsible for deciding if a submission should to to the publieke besluitendatabank based on a list of "rules". 

This PR does 2 things:

- update rules to include politiezone & hulpverleningszone submissions (https://github.com/lblod/prepare-submissions-for-export-service/pull/8)
- add migration to also export previously send in submission "retroactief" (this PR)

 ## Type of change


 ## How to test

- clone prepare-submissions-for-export service
- checkout the feature/additions-besluitenlijst branch
- then in app-digitaal-loket you have to link through volumes:
 ```
 environment: 
   NODE_ENV: "development"
  volumes:
   - "path/to/prepare-submissions-for-export/:/app/"
 ```
 
 - Next you need to link the consumer-producer from app-digitaal-loket to app-public-decisions-database
( Development setup guide here: https://github.com/lblod/app-digitaal-loket/wiki/Extending-Delta-Producer )
 
 - next login to frontend loket as a politieraad and submit a toezicht form
 - check the logs of the delate-consumer on the app-public-decisions-database and see if it starts consuming the besluit (logs will start moving a lot)

 ## Links to other PR's

 - prepare-submissions-for-export: https://github.com/lblod/prepare-submissions-for-export-service/pull/8
